[2023-12-03 22:12:54,426] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:12:54] "[35m[1mGET /filme?nome=Oppenheimer HTTP/1.1[0m" 500 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:13:17,588] ERROR log_exception() L1449 Exception on /filme [GET] - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\flask_openapi3\scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\pucpythontrab\backend_api_puc\app.py", line 83, in get_filme
    filme_id = query.id
               ^^^^^^^^
AttributeError: 'FilmeBuscaSchema' object has no attribute 'id'
[2023-12-03 22:13:18,059] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:13:18] "[35m[1mGET /filme?nome=1 HTTP/1.1[0m" 500 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:15:03,215] INFO _log() L224  [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:15:03,216] INFO _log() L224  [33mPress CTRL+C to quit[0m - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:15:04,715] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:15:04] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:15:04,835] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:15:04] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:15:04,838] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:15:04] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:15:04,839] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:15:04] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:15:04,985] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:15:04] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:15:04,993] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:15:04] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:15:08,967] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:15:08] "GET /filme?id=1 HTTP/1.1" 200 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:17:24,425] INFO _log() L224  [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:17:24,426] INFO _log() L224  [33mPress CTRL+C to quit[0m - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:17:26,507] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:17:26] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:17:26,627] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:17:26] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:17:26,628] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:17:26] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:17:26,630] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:17:26] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:17:26,743] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:17:26] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:17:26,744] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:17:26] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:17:30,277] WARNING get_filme() L96   Erro ao buscar o filme desejado.'Oppenheimer', Não há nenhum filme com esse nome no nosso banco de dados! - call_trace=C:\pucpythontrab\backend_api_puc\app.py L96  
[2023-12-03 22:17:30,278] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:17:30] "[33mGET /filme?nome=Oppenheimer HTTP/1.1[0m" 404 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:18:55,262] INFO _log() L224  [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:18:55,264] INFO _log() L224  [33mPress CTRL+C to quit[0m - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:18:57,225] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:18:57] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:18:57,335] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:18:57] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:18:57,336] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:18:57] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:18:57,336] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:18:57] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:18:57,410] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:18:57] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:18:57,411] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:18:57] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:18:59,204] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:18:59] "GET /filme?nome=Oppenheimer HTTP/1.1" 200 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:20:53,421] INFO _log() L224  [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:20:53,421] INFO _log() L224  [33mPress CTRL+C to quit[0m - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
[2023-12-03 22:20:54,855] INFO _log() L224  127.0.0.1 - - [03/Dec/2023 22:20:54] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\Intel\AppData\Roaming\Python\Python311\site-packages\werkzeug\_internal.py L224 
